cmd/sysl/cmd_diagram.go

```diff
@@ -26,9 +27,12 @@ func (p *diagramCmd) Configure(app *kingpin.Application) *kingpin.CmdClause {
 	cmd.Flag("sequencediagram",
 		"Generate a sequence diagram (Specify 'appname->endpoint')",
 	).Short('s').StringVar(&p.SequenceDiagram)
-	cmd.Flag("endpointanalysis", ""+
+	cmd.Flag("endpointanalysis",
 		"Generate an EPA diagram (Specify 'true')",
```

Comment:
Might want to be a bit more detailed about what an EPA diagram is?

Something like "Generate an integration diagram with endpoints displayed"